test/OpenTelemetry.Tests/Trace/ExportProcessorTest.cs

```diff
@@ -26,12 +25,13 @@ namespace OpenTelemetry.Tests.Trace
     public class ExportProcessorTest
     {
         private const string ActivitySourceName = "ActivityExportProcessorTest";
+        private List<Activity> exportedActivities = new List<Activity>();
```

Comment:
not blocking this PR, if the intention is to allow multiple test cases to run concurrently, consider using a thread safe version (e.g. concurrent collection)

Reply:
I believe xUnit creates a new instance of the class for each of the tests in the class. So these tests don't share the collection. I could move the instantiation to the constructor.